website/content/en/v0.7.0/AWS/provisioning.md

```diff
@@ -188,6 +188,18 @@ spec:
           throughput: 125
 ```
 
+### Context

```

Comment:
can you remove this and update the `preview` docs ? We store snapshot docs for the versions that are meant to be immutable so that users can view the docs at a specific version snapshot.  When a new release is made, the preview docs will become that version's snapshot.